### DISCUSSION TITLE
The Review Requested filter appears to be cached and not refreshing

### DISCUSSION THREAD

**Discussion by cpriestland** [community/community#44492] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

https://github.com/pulls/review-requested

Appears to be cached and not refreshing even though I have approved quite a few PR's it still displays the old ones. Not browser cache, happening to quite a few team members on different computers / networks.

**Comment by gszzzzzz** [comment]
It seems to be fixed for me now. In my case, the Pull Requests and Issues tabs were showing old(= cached) results, preventing them to be merged(for PRs) or closed(for Issues).

**Reply by cpriestland** [reply]
Still broken for me

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @cpriestland ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
